tests/core/test_expectation_suite.py

```diff
@@ -26,7 +33,290 @@ def exp1():
 
 
 @pytest.fixture
-def exp2():
+def expect_column_values_to_be_in_set_col_a_with_meta_dict() -> dict:
+    # Note, value_set is distinct to ensure this is treated as a different expectation
+    return {
+        "expectation_type": "expect_column_values_to_be_in_set",
+        "kwargs": {
+            "column": "a",
+            "value_set": [1, 2, 3, 4, 5],
+            "result_format": "BASIC",
+        },
+        "meta": {"notes": "This is an expectation."},
+    }
+
+
+class TestInitMethod:
+    """Tests related to ExpectationSuite.__init__()"""
```

Comment:
Just out of curiosity, what led you to use a class-based set of tests as opposed to individual functions? Still trying to learn how best to use this feature.

Reply:
I was thinking more about collecting tests related to specific methods in one place. It might be a little easier to see if a method is untested or not have to look far for related tests. Especially when new tests are added the hope is they get into the class and are not at the bottom of the file away from other related tests.
Anyway, this is new for me too with pytest but that was my thought process.